subsys/bluetooth/controller/hci_driver.c

```diff
@@ -1178,6 +1198,16 @@ static int configure_memory_usage(void)
 		CONFIG_BT_CTLR_SDC_ISO_TX_PDU_BUFFER_PER_STREAM_COUNT;
 #endif
 
+#if defined(CONFIG_BT_CTLR_SDC_CS_COUNT)
+	cfg.cs_count.count = CONFIG_BT_CTLR_SDC_CS_COUNT;
+	required_memory = sdc_cfg_set(SDC_DEFAULT_RESOURCE_CFG_TAG,
```

Comment:
since sdc_cfg_t is a union I think this overwrites/corrupts the ISO buffer configuration above (before it's passed to the controller in sdc_cfg_set() below)